File: VB/FilterLookUpsEditForm/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace FilterLookUpsEditForm
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnSingle_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnSingle.Click
			Dim frm As New FormSingleSource()
			frm.Show()
		End Sub

		Private Sub btnDiff_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnDiff.Click
			Dim frm As New FormDifferentSources()
			frm.Show()
		End Sub
	End Class
End Namespace

File: CS/FilterLookUpsEditForm/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace FilterLookUpsEditForm
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnSingle_Click(object sender, EventArgs e)
        {
            FormSingleSource frm = new FormSingleSource();
            frm.Show();
        }

        private void btnDiff_Click(object sender, EventArgs e)
        {
            FormDifferentSources frm = new FormDifferentSources();
            frm.Show();
        }
    }
}
